source/Preferences.cpp

```diff
@@ -37,6 +37,10 @@ namespace {
 	const string EXPEND_AMMO = "Escorts expend ammo";
 	const string FRUGAL_ESCORTS = "Escorts use ammo frugally";
 
+	const string DATEFMT = "Date format";
+	const vector<string> DATEFMT_OPTIONS = {"dmy", "mdy", "ymd"};
+	int dateFormatIndex = 1;
```

Comment:
```suggestion
	int dateFormatIndex = 0;
```

I don't think we should be changing the default :)

Reply:
Silly me. I was programming in Lua for a few hours and I forgot that C++ is zero-indexed, lol.